### ISSUE TITLE
Make column names consistent

### ISSUE THREAD

**Issue by glenn-m** [glenn-m/promR#18]
Currently we have differing column names depending on query, we should really make it more consistent. Mainly the `name` column name. 

e.g. 

An instant query results in:

| X\_\_name\_\_  | instance                 | job          | timestamp      | value | port |
| :------------- | :----------------------- | :----------- | :------------- | :---- | ---: |
| go\_goroutines | demo.robustperception.io | prometheus   | 1554235843.452 | 87    | 9090 |
| go\_goroutines | demo.robustperception.io | pushgateway  | 1554235843.452 | 40    | 9091 |
| go\_goroutines | demo.robustperception.io | alertmanager | 1554235843.452 | 34    | 9093 |
| go\_goroutines | demo.robustperception.io | node         | 1554235843.452 | 8     | 9100 |

where as range query results in: 

| **name**       | instance                 | job        | port | timestamp      | value |
| :------------- | :----------------------- | :--------- | ---: | :------------- | :---- |
| go\_goroutines | demo.robustperception.io | prometheus | 9090 | 1554235303.644 | 85    |
| go\_goroutines | demo.robustperception.io | prometheus | 9090 | 1554235313.644 | 85    |
| go\_goroutines | demo.robustperception.io | prometheus | 9090 | 1554235323.644 | 85    |
| go\_goroutines | demo.robustperception.io | prometheus | 9090 | 1554235333.644 | 85    |
| go\_goroutines | demo.robustperception.io | prometheus | 9090 | 1554235343.644 | 85    |
| go\_goroutines | demo.robustperception.io | prometheus | 9090 | 1554235353.644 | 85    |

tl;dr `X__name__` should be `name` instead

**Comment by konradzdeb** [comment]
I'm wondering whether this is because of the `make.names` being called somewhere. I reckon that if the original `name` is not syntactically correct, `as.data.frame` will force **`X__`** as in:
```r
make.names("1__name__")
#[1] "X1__name__"
```r

could be easily withed with some basic string operations.
